### DISCUSSION TITLE
Distributed Deployment

### DISCUSSION THREAD

**Discussion by Kryptoniteee** [Security-Onion-Solutions/securityonion#3481] [Unsupported Versions]
Hi my company is a mssp hoping to host security onion and monitor my client's network for them.
Referring to the distributed deployment architecture, how exactly are the forward nodes going to integrate with my client's syslog server and also have their logs to be sent over from their network to my network which hosts the security onion manager and search nodes?

Any advice?

**Comment by weslambert** [comment]
You can send syslog to a forward node, and the events will be forwarded to the manager, then processed by the downstream search nodes.  If you are sending from site-to-site, we recommend using a VPN solution like OpenVPN.
